README.md

```diff
@@ -20,7 +20,7 @@ elm-json uninstall elm-community/array-extra elm-community/basics-extra elm-comm
 
 elm make src/Main.elm # or whatever else you use to build your application
 # There shouldn't be any errors, but if there somehow are any, now would be a good time to fix them.
-elm-test # Also, should work exactly the same. Please report any issues.
+elm-test # should also work exactly the same. Please report any issues.
```

Comment:
```suggestion
elm-test # Should also work exactly the same. Please report any issues.
```

Reply:
all other comments that follow a command on the same line start lowercase. Any reason this one's special?

Reply:
We should pick one or the other. I'd go for sentences for prose.